andela/ah-backend-stark
# #162082219 fix followers list

#### What does this PR do?
Fixes the endpoint that retrieves a list of followers
#### Description of Task to be completed?
ensure that the following endpoint is working:
`GET api/profile/<str:username>/followers/`
#### How should this be manually tested?
- After pulling this branch locally, run `python manage.py runserver`
- Open Postman and create a user using the following endpoint
`POST api/user/'
`GET api/users/activate_account/<token>/` (use the token from the registration response)
- Create another user and follow the first user using the following endpoint
`POST api/profile/<str:username>/follow/`
- Check that user's followers using the following endpoint
`GET api/profile/<str:username>/followers/`
#### Any background context you want to provide?
The bug was introduced while refactoring the work.
#### What are the relevant pivotal tracker stories?
[#162082219](https://www.pivotaltracker.com/story/show/162082219)
#### Screenshots
![image](https://user-images.githubusercontent.com/40589388/48771101-69136980-ecd1-11e8-906d-664d03417c6a.png)
